sc-memory/tests/sc-memory/events/test_sc_event.cpp

```diff
@@ -408,3 +408,49 @@ TEST_F(ScEventTest, pend_events)
 
   EXPECT_EQ(passedCount, el_num);
 }
+
+TEST_F(ScEventTest, BlockEvents)
```

Comment:
Add test where event is emitted. And where one event is emitted and after blocking not emitted